src/graphics/webgl/webgl-buffer.js

```diff
@@ -0,0 +1,52 @@
+import { BUFFER_DYNAMIC, BUFFER_GPUDYNAMIC, BUFFER_STATIC, BUFFER_STREAM } from '../constants.js';
+
+/**
+ * A WebGl implementation of the Buffer.
```

Comment:
I suspect that this will appear in the API reference manual unless you add an `@ignore` tag.

Reply:
And WebGL has a capital 'L'. 😄

Reply:
fixed!, thansk